core/src/test/java/org/apache/brooklyn/entity/group/DynamicClusterTest.java

```diff
@@ -1398,14 +1398,18 @@ public void testChildCommandPermitNotReleasedWhenMemberStartTaskCancelledBeforeS
     public void testClusterMaxSize() {
```

Comment:
If we're changing the `shrink` behaviour (see previous comment), then a corresponding `testClusterMinSize` should also be added